core/includes/config.inc

```diff
@@ -335,8 +335,16 @@ function config_uninstall_config($module, $config_name = NULL) {
  *   A ConfigStorageInterface object managing the specified configuration type.
  */
 function config_get_config_storage($type = 'active') {
```

Comment:
Right now I believe sites will break if they don't specify these settings. We should not require people to update their settings files to keep the sites working, so there should be a fallback if there is no setting.

Reply:
Oh, there is a default.